src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -146,9 +148,16 @@ public enum DesiredLocation {
   private DesiredLocation desiredLocation = DesiredLocation.Reef0;
 
   private boolean isOTF = false;
-
   private Command driveToPose = null;
 
+  private final ProfiledPIDController angleController =
+      new ProfiledPIDController(5, 0.0, 0.4, new TrapezoidProfile.Constraints(1000, 2000));
```

Comment:
Can you add these gains to a JSON so we can configure them more easily?

Also 1000 and 2000 seem like insane choices for constraints. I think we should probably pick some real values here